lib/fastlane/plugin/yarn/actions/yarn_action.rb

```diff
@@ -2,18 +2,25 @@ module Fastlane
   module Actions
     class YarnAction < Action
       def self.run(params)
-        task = params[:task]
+        flags = params[:flags]
         command = params[:command]
+        options = params[:options]
         package_path = params[:package_path]
 
         # create a new object from yarn helper
         yarn = Helper::YarnHelper.new(package_path: package_path)
 
         # Check if yarn is installed
         yarn.check_install
+        yarn.install_dependencies
```

Comment:
Yeah I was also, it does seem wasteful to keep re-installing if using `yarn` in multiple places in your `Fastfile`.